buildman/manager/executor.py

```diff
@@ -215,10 +215,14 @@ def _get_conn(self):
 
     @property
     def running_builders_count(self):
-        ec2_conn = self._get_conn()
-        resp = ec2_conn.describe_instances(
-            Filters=[{"Name": "tag:Name", "Values": ["Quay Ephemeral Builder"]}]
-        )
+        try:
```

Comment:
Any reason you decided to put this try/except statement at this level, instead of within the `_get_conn()` method?

Reply:
I did overlook that this also catches any error which might occur when hitting the API to fetch the instances. That make sense to be covered here.

Reply:
Doesn't really make sense to try/catch this in `_get_conn`, since we're trying to catch client errors on the api call. `_get_conn` would not raise any exception.

Reply:
:+1: